packages/fxa-admin-panel/src/lib/config.ts

```diff
@@ -0,0 +1,84 @@
+// This configuration is a subset of the configuration declared in server/config/index.ts
+// Which config is copied over is defined in server/lib/server.ts
```

Comment:
Sounds like `Pick` (https://www.typescriptlang.org/docs/handbook/utility-types.html#picktk) can be used here.

Reply:
@chenba Hmm I'm having trouble with specifically this change (my other switches to Pick worked fine).

New setup looks like:

```js
import { default as ServerConfig } from '../../server/config';

export type Config = Pick<ServerConfig, 'env' | 'servers'>;
```

But TS is not liking this: `'ServerConfig' refers to a value, but is being used as a type here.`

Any idea how I can address this? I would have thought the types on `ServerConfig` would come through.

Reply:
If the type is exported then import the type.  Otherwise, I think `Pick<typeof ServerConfig, ...` should work.

Reply:
Ah ok that did it, but that seems to have surfaced a new thing. Although the note indicates it's a subset of the noted config file, I don't think I can `Pick` from it because ServerConfig is an instance of Convict that relies on get and set methods to retrieve properties, while this new config is just a function that returns a standard object. Unless I'm missing something 🤔

Reply:
Oh ok.  My comment was based on that code comment, so I didn't know it was a value.